autoload/db_ui/dbout.vim

```diff
@@ -52,7 +55,7 @@ function! db_ui#dbout#get_cell_value() abort
   endif
 
   let cell_line_number = s:get_cell_line_number(scheme)
-  let cell_range = s:get_cell_range(getline(cell_line_number), col('.'))
+  let cell_range = s:get_cell_range(cell_line_number, getcurpos(), parsed.scheme)
```

Comment:
I started using `getcurpos()` here rather than `col('.')`. The reason is that `getcurpos()` tracks the "virtual" column which is desired for vertical movement of a cursor. This helps alleviate the issues we were facing with tabs causing the column number to be different depending on the line, since tabs can account for any number of characters up until the `&tabstop`.

Reply:
There are a lot of changes in `s:get_cell_range` function, and I'm not sure how it will reflect other databases. I don't want to break anything that already works. Let's have a separate function for `oracle` to get the cell range and do all logic for it separately, and restore this function to do what it did before.

Reply:
I'll implement an `if` inside `get_cell_range` to call the new function if the `scheme_name` is oracle. That way if any new DBMS suites are released which need the new function, we can implement the detection in one place

Reply:
I also am wondering if we can shortcut this process entirely by setting `expandtab` on dbout.